src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2837,6 +2847,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         e.DragUIOverride().IsGlyphVisible(false);
     }
 
+    // Method description:
+    // - Checks if the uri is valid and sends an event if so
+    // Arguments:
+    // - The uri
+    void TermControl::_HyperlinkHandler(std::wstring uri)

```

Comment:
for string parameters where we don't need to take a copy, prefer `const std::wstring_view` (or string_view)